README.md

```
@@ -11,9 +11,9 @@ By default the project is not configured to run any test suite: an attempt to ex
 ### Local test execution
 | Test Suite            | Command |
 |-----------------------|---------|
-| REST API tests        | macOS/Linux:<br/><pre>./gradlew runStories -Pvividus.configuration.suites=rest_api</pre><br/>Windows:<br/><pre>gradlew runStories -Pvividus.configuration.suites=rest_api</pre> |
+| REST API tests        | macOS/Linux:<br/><pre>./gradlew runStories -Pvividus.configuration-set.active=rest-api</pre><br/>Windows:<br/><pre>gradlew runStories -Pvividus.configuration-set.active=rest-api</pre> |
 ||
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)